Update PrivateToken.parse to be synchronous

thibmeu opened the pull request:
Both PrivateToken.parse and PrivateToken.parseMultiple are not awaiting on any Promise. They don't need to be async.

This commit updates these function signatures to reflect this.

thibmeu commented:
Updated the PR to include `PrivateToken.toString` as well. No need for asynchronicity.

armfazh commented on test/authScheme.test.ts:
```diff
@@ -41,7 +41,7 @@ test.each(tokenVectors)('AuthScheme-TokenVector-%#', async (v: TokenVectors) =>
 type HeaderVectors = (typeof headerVectors)[number];
 
 test.each(headerVectors)('AuthScheme-HeaderVector-%#', async (v: HeaderVectors) => {
```
```suggestion
test.each(headerVectors)('AuthScheme-HeaderVector-%#', (v: HeaderVectors) => {
```

thibmeu replied:
updated

thibmeu committed:
Update PrivateToken.parse to be synchronous

Both PrivateToken.parse and PrivateToken.parseMultiple are not awaiting
on any Promise. They don't need to be async.

This commit updates these function signatures to reflect this.
